test/test_elements.jl

```diff
@@ -110,19 +110,19 @@ end
 
     # matrix and edges
     let x = randn(10), y = randn(5), z = cos.(x .+ y')
-        @test pgf.Table(x, y, z) ≅ pgf.Table(pgf.Options(),
-                                             hcat(pgf.matrix_xyz(x, y, z)...),
+        @test Table(x, y, z) ≅ Table(PGFPlotsX.Options(),
+                                             hcat(PGFPlotsX..matrix_xyz(x, y, z)...),
```

Comment:
looks like an extra `.` here

Reply:
Aaargh, it is impossible to use the online editor. Never agian.